lib/mshoplib/src/MShop/Plugin/Provider/Order/PropertyAdd.php

```diff
@@ -157,9 +158,21 @@ protected function addAttributes( \Aimeos\MShop\Order\Item\Base\Product\Iface $o
 			return $orderProduct;
 		}
 
+		if($products->count() > 1) {
+			$variant = $products->find(
+				function ( \Aimeos\MShop\Product\Item\Iface $item ) use ( $orderProduct ) : bool {
+					return $item->getCode() === $orderProduct->getProductCode();
+				}
+			);
+		}
+
```

Comment:
It's more efficient if you use `$map = $products->col( null, 'product.code' );` first and then access the products by code

Reply:
Nice! Im used to using laravel collections and find myself digging through your docs for functions that are similar that I can use.

Reply:
The Map methods are very close to Laravel, only if standard PHP uses another name (`array_column()` instead of Laravel's `pluck()`), then the Map methods follow the PHP naming.